app/models/vacancy.rb

```diff
@@ -100,7 +94,7 @@ class Vacancy < ApplicationRecord
                   if: proc { |vacancy| vacancy.listed? }
 
   before_save :on_expired_vacancy_feedback_submitted_update_stats_updated_at
-  before_save :on_job_role_set_job_roles
+  before_save :on_job_roles_set_job_role
```

Comment:
We went from populating `job_roles` on DB when the `job_role` is set from user flow to doing the opposite: 
Now the user flow will populate `job_roles`, and the service will map it to `job_role` in DB.